lstgen/generators/go.py

```diff
@@ -0,0 +1,268 @@
+"""
+Go generator
+"""
+import ast
+from .. import (
+    parse_eval_stmt,
+    prepare_expr
+)
+from .. import (
+    EvalStmt,
+    IfStmt,
+    ThenStmt,
+    ElseStmt,
+    ExecuteStmt,
+)
+from .base import JavaLikeGenerator
+
+class GoGenerator(JavaLikeGenerator):
+    """ Java Generator """
+    bd_class = 'decimal.Decimal'
+    bd_class_constructor = 'decimal.NewFromInt'
+    list_const_parens = ('{', '}')
+    stmt_separator = ''
+    allow_constants = False
+
+    def __init__(self, parser, outfile, class_name=None, indent=None, package_name='default'):
+        super(GoGenerator, self).__init__(parser, outfile, class_name, indent)
+        self.package_name = package_name
+
+    def generate(self):
+        wr = self.writer
+        self._write_comment("This file is automatically generated by LstGen, do not edit!", False)
+        wr.writeln('package {}'.format(self.package_name or 'tax'))
```

Comment:
CLI suggests the package name is "default"

Reply:
"default" is a reserved keyword in Go, so that can not be fixed without breaking compatibility

Reply:
For Golang, you should make an exception here, because it makes no sense to call a package "default". It should describe the "purpose" or "function" of the module. So "tax" makes more sense here, even if I would expect something like 'LstGen' or something similar.

Reply:
Let's just fix the CLI docs then: https://github.com/jenner/LstGen/pull/15/files#diff-85770ee9cd547a957b0df5b6043486d3R93

Reply:
Done! :-)